compiler/lib/src/main/scala/codegen/CppWriter/TypeCppWriter.scala

```diff
@@ -16,6 +17,10 @@ case class TypeCppWriter(
     override def absType(s: CppWriterState, t: Type.AbsType) =
       s.writeSymbol(Symbol.AbsType(t.node))
 
+    override def aliasType(s: CppWriterState, t: Type.AliasType) =
+      // TODO(tumbar) Keep the aliased type name in codegen
```

Comment:
Please clarify that we're doing it this way for now because we're not generating the type alias code.